### PR TITLE
Selection

### PR DESCRIPTION
**Pull request by codingoak**
Issue 4, Sprint 1

### CODE REVIEW

File: src/components/Issues.spec.js [hunk 1]
```diff
@@ -7,8 +7,8 @@ describe('Issues', () => {
     render(
       <Issues
         savedIssues={[
-          { id: 111, title: 'Teststring1', state: 'open', clicked: true },
-          { id: 112, title: 'Teststring2', state: 'close', clicked: false },
+          { id: 111, title: 'Teststring1', state: 'open', sPinned: true },
```
Comment (Quaquaro) [inline src/components/Issues.spec.js:10]: Looks like you forgot the i in isPinned

File: src/pages/Dashboard.js [hunk 2]
```diff
@@ -10,43 +10,33 @@ export default function Dashboard({
...
+        <>
+          {/* <Selection /> */}
+          <Issues savedIssues={savedIssues} togglePin={togglePin} />
+        </>
       )}
       {hasError && (
         <ErrorContainer>
           <ErrorState>Oops, something went wrong</ErrorState>
-          <Button handleClick={resetError}>TRY AGAIN</Button>
+          <Button handleClick={() => GetFetch(selectedProject)}>
```
Comment (Quaquaro) [inline src/pages/Dashboard.js:31]: Nice One!

**Review by Quaquaro — APPROVED** [review]
Great job! See you later!

File: src/components/Issues.spec.js (hunk 1 above)
Comment (codingoak) [reply src/components/Issues.spec.js:10]: Oh yes thx - fixed

**Pull request merged**